app/steps/ui/tasklist/includes/eligibility.html

```diff
@@ -1,18 +1,29 @@
 {% set previousTaskStatus = fields.previousTaskStatus.value.EligibilityTask %}
 
 <li class="task-item">
-    <span class="heading-medium">{{content.eligibilityTask}}</span>
-    {% if fields.EligibilityTask.value.status == 'complete' and previousTaskStatus !== 'started' %}
-        <span class="progress-status-tag">{{content.taskCompleteTag | upper}}</span>
-        <p>{{content.eligibilityTaskCompleteParagraph1 | safe}}</p>
+
+    {{taskStatus(
+        content = content,
+        title = content.eligibilityTask,
+        status = fields.EligibilityTask.value.status,
+        previousTaskStatus = 'complete'
+    )}}
+
+    {% if fields.EligibilityTask.value.status == 'complete' %}
+        <div class="task-item-content">
```

Comment:
This wrapper should sit outside the `if` statement so it doesn't need to be defined in each part of the `if`/`else` statement. 

However as we're now using `.task-item-header` for clearing the title.tag floats we probably don't need it any longer.

The same comment applies to all instances of `.task-item-content`.

Reply:
I've only changed the files that have both an if/else statement, otherwise if there is only an if statement this results in an empty <div>